### PR TITLE
CONN-1254 Updates to UI and other improvements

### PR DESCRIPTION
**Pull request by jsmith2**
Do not merge until Kerry has reviewed.

Includes fix for delete ajax and for database errors and for clearing address fields after add.

### CODE REVIEW

**Comment by kerry-chin** [comment]
Couple of comments:

(1) directPrime.xhtml -- this file doesn't have the fixes that move all action buttons to underneath the tables.

(2) We can remove the following folder (and their fonts) from the PR:
--> Product/Production/Adapters/General/CONNECTAdminGUI/src/main/webapp/fonts/

**Comment by jsmith2** [comment]
This has been reviewed and updated by Kerry and is ready for merge.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/direct/DirectDomainBean.java [hunk 1]
```diff
@@ -458,31 +452,19 @@ protected void resetBean() {
...
+    public void setAddressEmail(String addressEmail) {
+        this.addressEmail = addressEmail;
     }
 
-    /**
-     * @param displayName the displayName to set
-     */
-    public void setDisplayName(String displayName) {
-        this.displayName = displayName;
+    private boolean notNullish(String value){
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/direct/DirectDomainBean.java]: There's a static "isNullish" function in CoreLib's "gov.hhs.fha.nhinc.nhinclib.NullChecker" we could leverage for this.

File: Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/direct/DirectServiceImpl.java [hunk 2]
```diff
@@ -23,21 +23,15 @@
 import gov.hhs.fha.nhinc.admingui.proxy.DirectConfigProxy;
 import gov.hhs.fha.nhinc.admingui.services.DirectService;
 import java.util.List;
+import java.util.logging.Level;
```
Comment (christophermay07) [inline Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/services/impl/direct/DirectServiceImpl.java]: Unused import.

**Pull request merged**